taxi/remote.py

```diff
@@ -96,7 +96,7 @@ def _login(self):
         else:
             self.logged_in = True
 
-    def send_entries(self, entries, mappings, callback=None):
+    def send_entries(self, entries, mappings, callback=None, skip_today=False):
```

Comment:
I don't think skipping entries should be done in the remote. This should be done in the command instead.